examples/erc20Dividend.ts

```diff
@@ -0,0 +1,193 @@
+import { BigNumber } from '@0x/utils';
+import { RedundantSubprovider, RPCSubprovider, Web3ProviderEngine } from '@0x/subproviders';
+
+import { ERC20DividendCheckpointEvents } from '@polymathnetwork/abi-wrappers/lib/src';
+import { ApiConstructorParams, PolymathAPI } from '../src/PolymathAPI';
+import { bytes32ToString } from '../src/utils/convert';
+import { ModuleName, ModuleType } from '../src';
+import ModuleFactoryWrapper from '../src/contract_wrappers/modules/module_factory_wrapper';
+
+// This file acts as a valid sandbox for adding a etherDividend  module on an unlocked node (like ganache)
+
+window.addEventListener('load', async () => {
+  // Setup the redundant provider
+  const providerEngine = new Web3ProviderEngine();
+  providerEngine.addProvider(new RedundantSubprovider([new RPCSubprovider('http://127.0.0.1:8545')]));
+  providerEngine.start();
+  const params: ApiConstructorParams = {
+    provider: providerEngine,
+    polymathRegistryAddress: '<Deployed Polymath Registry address>',
+  };
+
+  // Instantiate the API
+  const polymathAPI = new PolymathAPI(params);
+
+  // Get some poly tokens in your account and the security token
+  const myAddress = await polymathAPI.getAccount();
+  // Token faucet on test net only
+  await polymathAPI.getPolyTokens({ amount: new BigNumber(1000000), address: myAddress });
+  await polymathAPI.polyToken.transfer({
```

Comment:
Same as above. You don't need to transfer. You only need to approve as you are doing below, then the ST calls `transferFrom`. So this entire line is not needed.